types/react-onsenui/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for React Onsen UI (react-onsenui) 2.8
+// Type definitions for React Onsen UI (react-onsenui) 2.9
```

Comment:
Do not increase version as the latest Onsen UI is currently 2.8. See https://medium.com/the-web-tub/new-in-onsen-ui-2-8-app-size-is-much-smaller-than-before-d2b99e594aef